.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+profile=black
+known_first_party=['ansiblelint']
+known_third_party = ansible,pytest,ruamel,setuptools,sphinx,yaml
```

Comment:
Sorry but no. It would introduce additional complexity which is not needed, or standard. If we are to do stuff like this they would have to be done totally separated from this, with a ticket and consultation with others regarding if pros/cons.

The only reason why I added the config is because isort needed some help detecting the layout and not for introducing custom standards.

Reply:
There's no description in the commit message https://github.com/ansible/ansible-lint/commit/9d08973199dbabdb8c070e405c8548905d145294.

Reply:
There is now